extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -97,21 +99,16 @@
   right: 0 !important;
   top: 0 !important;
   height: 56px !important;
+  z-index: 1 !important;
   background: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0));
-  z-index: 2 !important;
 }
 
 @media (min-width: 1024px) {
   .i-amphtml-lbg-top-bar {
-    height: 100px !important;
```

Comment:
Corrected typo here.

Reply:
Nice, if two values need to match in CSS it can be helpful to add a reminder comment.